### ISSUE TITLE
About the training of Camvid.

### ISSUE THREAD

**Issue by sunpeng1996** [Reagan1311/DABNet#13]
Hi, thanks for your excellent work.
I have a question, the Camvid dataset is small (i,e. the images are few), do you share your training tricks or tips for Camvid?

And I see you that your max-epoch is 1000 on Camvid, don't you? Wow, it's too many epochs.
Thanks!!!

**Comment by Reagan1311** [comment]
Adam is more stable in the low-resolution image like CamVid dataset and class weights are employed in the loss function just as ENet.
Actually, it does not need to set the epoch to 1000 on camvid, the model can saturate about 500~800 epochs, I just set it the same as cityscapes, you can try different epoch.
